docs/en/compute/AWS-EMR.md

```diff
@@ -0,0 +1,107 @@
+---
+layout: global
+title: Running Alluxio on EMR
+nickname: AWS EMR
+group: Data Applications
+priority: 0
+---
+
+This guide describes how to configure Alluxio to run on [AWS EMR](https://aws.amazon.com/emr/).
+
+* Table of Contents
+{:toc}
+
+## Overview
+
+AWS EMR provides great options for running clusters on-demand to handle compute workloads. It manages the
+deployment of various Hadoop Services and allows for hooks into these services for customizations. Alluxio
+can run on EMR to provide functionality above what EMRFS currently provides. Aside from the added performance
+benefits of caching, Alluxio also enables users to run compute workloads against on-premise storage or even a
+different cloud provider's storage i.e. GCS, Azure Blob Store. 
+
+## Prerequisites
+
+* Account with AWS
+* IAM Account with the default EMR Roles
+* Key Pair for EC2
+* An S3 Bucket
+* AWS CLI
+
+The majority of the pre-requisites can be found by going through the
+[AWS EMR Getting Started](https://docs.aws.amazon.com/emr/latest/ManagementGuide/emr-gs.html) guide. An S3 bucket
+is needed as Alluxio's Root Under File System. If required, this can be reconfigured to be HDFS if needed.
+
+## Basic Setup
+
+To begin with, download the alluxio-emr.sh and alluxio-presto.json files from Github. These files will serve as the
```

Comment:
-  `` `alluxio-emr.sh` `` also `` `alluxio-presto.json` ``.
- we shall push them to alluxio repo too
- what about spark? do we need a alluxio-spark.json in the future?

Reply:
Didn't add the link because this PR needs to be merged before the link will resolve to a file.
I think all the service JSON's can be merged into one, but it will require more investment. I think we should start with Presto and then make modifications on top.